docs/tasks/run-application/configure-pdb.md

```diff
@@ -0,0 +1,225 @@
+of the number of desired replicas are healthy. 
+
+Example 3: With a `maxUnavailable` of 5, evictions are allowed as long as there are at most 5
+unhealthy replicas among the total number of desired replicas.
+
+Example 4: With a `maxUnavailable` of 30%, evictions are allowed as long as no more than 30% 
+of the desired replicas are unhealthy.
+
+In typical usage, a single budget would be used for a collection of pods managed by
+a controller—for example, the pods in a single ReplicaSet or StatefulSet. 
+
+Note that a disruption budget does not truly guarantee that the specified
+number/percentage of pods will always be up.  For example, a node that hosts a
+pod from the collection may fail when the collection is at the minimum size
+specified in the budget, thus bringing the number of available pods from the
+collection below the specified size. The budget can only protect against
+voluntary evictions, not all causes of unavailability.
+
+A `maxUnavailable` of 0% (or 0) or a `minAvailable` of 100% (or equal to the
+number of replicas) may block node drains entirely. This is permitted as per the 
+semantics of `PodDisruptionBudget`.
+
+You can find examples of pod disruption budgets defined below. They match pods with the label 
+`app: zookeeper`.
+
+Example PDB Using maxUnavailable:
+
+```yaml
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: zk-pdb
+spec:
+  minAvailable: 2
+  selector:
+    matchLabels:
+      app: zookeeper
+```
+
+Example PDB Using maxUnavailable (Kubernetes 1.7 or higher):
+
+```yaml
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: zk-pdb
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: zookeeper
+```
+
+For example, if the above `zk-pdb` object selects the pods of a StatefulSet of size 3, both
+specifications have the exact same meaning. The use of `maxUnavailable` is recommended as it
+automatically responds to changes in the number of replicas of the corresponding controller.
+
+# Create the PDB object
+
+You can create the PDB object with a command like `kubectl create -f mypdb.yaml`.
+
+You cannot update PDB objects.  They must be deleted and re-created.
+
+# Check the status of the PDB
+
+Use kubectl to check that your PDB is created.
+
+Assuming you don't actually have pods matching `app: zookeeper` in your namespace,
+then you'll see something like this:
+
+```shell
+$ kubectl get poddisruptionbudgets
+NAME      MIN-AVAILABLE   ALLOWED-DISRUPTIONS   AGE
+zk-pdb    2               0                     7s
+```
+
+If there are matching pods (say, 3), then you would see something like this:
+
+```shell
+$ kubectl get poddisruptionbudgets
+NAME      MIN-AVAILABLE   ALLOWED-DISRUPTIONS   AGE
+zk-pdb    2               1                     7s
+```
+
+The non-zero value for `ALLOWED-DISRUPTIONS` means that the disruption controller
+has seen the PDB and counted the matching PDB, and updated the status
+of the PDB.  
```

Comment:
this is confusing ... maybe you mean ... the disruption controller has seen the pods, and counted the matching pods, and update the status of the PDB.